search-commons/src/main/java/no/unit/nva/search2/model/OpenSearchQuery.java

```diff
@@ -127,11 +125,7 @@ public boolean hasNoSearchValue() {
     public void setSearchFieldValue(K key, String value) {
         if (nonNull(value)) {
             var decodedValue = key.valueEncoding() != ValueEncoding.NONE ? decodeUTF(value) : value;
-            if (key.fieldType() == ParamKind.STRING) {
```

Comment:
query values should not be modified, only when values are transformed into query